### DISCUSSION TITLE
請教Merge怎麼正確加網址令它走代理？

### DISCUSSION THREAD

**Discussion by YheonYeung** [libnyanpasu/clash-nyanpasu#1034] [Q&A] [answered]
我是用開CFW，有加預處理的，剛用喵帕斯。
嘗試登入這個網，是彈不出驗證的。提示網路異常
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/63342274/d4b1ae6c-e725-4e82-b5bd-3198f667f12e)

我看了這個網的，試著添加hoyoverse.com
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/63342274/75091c5e-3322-4e8f-a329-c6c328f4b232)
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/63342274/d529438b-1134-4f15-8bfd-bd95ddc19c0a)
也啟用了Merge
![image](https://github.com/LibNyanpasu/clash-nyanpasu/assets/63342274/f043801d-09a8-4be5-8983-e6400f4e900f)

請教下哪裡有不對，依然登不上。
另外希望nyanpasu.elaina.moe上面加個示例。方便我這種不會搞的可以抄個作業。

**Comment by keiko233** [comment]
按照clash的rule格式编写即可，wiki这部由于缺少编辑者，更新会较慢

**Comment by YheonYeung** [comment]
ehh...但就是上不去。全局就可以。寫在append-rules沒錯吧？

**Reply by keiko233** [reply, marked as answer]
https://nyanpasu.elaina.moe/zh-CN/tutorial/chians/custom-rules

文档已更新

**Reply by YheonYeung** [reply]
> https://nyanpasu.elaina.moe/zh-CN/tutorial/chians/custom-rules
> 
> 文档已更新
感謝！啊原來我寫錯。啊哈

**Comment by YheonYeung** [comment]
![2024-07-24_014633](https://github.com/user-attachments/assets/7f3d1e99-514b-461d-a272-8eb6f93d0700)
請教！
請問我想讓8.211.XXX.XXX這個IP段走代理，要怎麼寫才正確？
我試了「IP-CIDR,8.211.156.120/12,🚀 节点选择」「IP-CIDR,8.208.0.0/12,🚀 节点选择」。「/12」我也不懂什麼意思，照抄的。
都無法成功讓它走代理。

**Comment by YheonYeung** [comment]
@keiko233 
打擾了。
上面的新問題找到原因了，似乎就是這個只讀的配置已經有規則。
但是這個文件怎麼改呀？記事本打開刪掉，一開clash它又恢復了
![image](https://github.com/user-attachments/assets/a2a0b788-6714-47d0-ac49-3fa8db455ee0)

**Reply by keiko233** [reply]
运行时配置不可修改，如果需要修改需要修改订阅中的配置文件。
或者用script直接对规则进行修改，防止后期更新覆盖

**Reply by YheonYeung** [reply]
> 运行时配置不可修改，如果需要修改需要修改订阅中的配置文件。 或者用script直接对规则进行修改，防止后期更新覆盖

感謝解答。
原來是訂閱自帶的呀

![image](https://github.com/user-attachments/assets/b6432477-1bd9-483b-baa8-d0870534c1e9)
刪掉了訂閱裡面的。merge也有  - IP-CIDR,8.208.0.0/12,🚀 节点选择 - IP-CIDR,8.211.0.0/12,🚀 节点选择。還是DIRECT
